docs/timezone.md

```diff
@@ -8,13 +8,68 @@
+Each time zone has a canonical identifier, and may also have one or more aliases.
+
+Anywhere in ECMAScript that a `Temporal.TimeZone` object is expected, a time zone identifier can be used instead.
+Furthermore, using string identifiers instead of time zone objects allows ECMAScript implementations to perform optimizations that are not possible when using time zone objects.
+Therefore, it's recommended to always use string identifiers instead of time zone objects whenever this is convenient. For example:
+
+```javascript
+inBerlin = Temporal.ZonedDateTime.from('2022-01-28T19:53+01:00[Europe/Berlin]');
+inTokyo = inBerlin.withTimeZone('Asia/Tokyo'); // May be faster / use less RAM
+inTokyo = inBerlin.withTimeZone(Temporal.TimeZone.from('Asia/Tokyo')); // OK, but not optimal
+```
+
+### Canonicalization of time zone identifiers
+
+The `id` property of any `Temporal.TimeZone` instance returns the canonical identifier for that time zone, even if the input used to create that instance used an alias and/or a different mix of upper and lower case letters. For example:
+
+```javascript
+// In Firefox 109, India's time zone's canonical identifier is 'Asia/Kolkata'
+Temporal.TimeZone.from('Asia/Kolkata').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('Asia/Calcutta').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('asia/calcutta').id; // => 'Asia/Kolkata'
+Temporal.TimeZone.from('ASIA/CALCUTTA').id; // => 'Asia/Kolkata'
+
+// In Chrome 109, India's time zone's canonical identifier is 'Asia/Calcutta'
+Temporal.TimeZone.from('Asia/Kolkata').id; // => 'Asia/Calcutta'
+Temporal.TimeZone.from('Asia/Calcutta').id; // => 'Asia/Calcutta'
+
+// "UTC" is the canonical identifier for all UTC time zone aliases.
+Temporal.TimeZone.from('Etc/UTC').id; // => 'UTC'
+Temporal.TimeZone.from('Etc/GMT').id; // => 'UTC'
+Temporal.TimeZone.from('UTC').id; // => 'UTC'
+
+// Offset time zone identifiers are resolved to a canonical format
+Temporal.TimeZone.from('01').id; // => '+01:00'
+Temporal.TimeZone.from('+01').id; // => '+01:00'
+Temporal.TimeZone.from('+01:00').id; // => '+01:00'
+Temporal.TimeZone.from('+0100').id; // => '+01:00'
+Temporal.TimeZone.from('+01:00:00').id; // => '+01:00'
+```
+
+The canonical identifier for a time zone is guaranteed to be the same for the lifetime of any `Temporal`-using program.
+However, the canonical identifier is not guaranteed to be the same across time or between different ECMAScript implementations.
+To verify that two built-in time zone identifiers represent the same time zone, use `Temporal.TimeZone.from` to create `Temporal.TimeZone` instances, and then compare the `id` of both instances.
+
+```javascript
+function isSameTimeZone(id1, id2) {
+  return Temporal.TimeZone.from(id1).id === Temporal.TimeZone.from(id2).id;
+  // DON'T DO THIS: return id1 === id2;
+}
+areTimeZoneIdentifiersEquivalent('Asia/Calcutta', 'ASIA/KOLKATA'); // => true
+```
```

Comment:
This would need to be replaced by something like the below code, right?
```js
function isSameTimeZone(id1, id2) {
  return Temporal.TimeZone.from(id1).equals(id2);
  // DON'T DO THIS: return id1 == id2;
}
isSameTimeZone('Asia/Calcutta', 'ASIA/KOLKATA');  // => true
```

Reply:
Correct.